README.md

```diff
@@ -89,6 +89,16 @@ var opts = {
 }
 
 logger.log('My Sample Log Line', opts);
+
+// or pass any associated  object or a string along all lines of a logger
+// note: it will parse only two levels of the object
```

Comment:
`// Note: objects are only parsed two levels deep`

Reply:
thank you!